tests/functional/generate_pages.py

```diff
@@ -69,6 +69,13 @@
 
 """
 
+RELATIONSHIP_RADIO_CLICKER = r"""  click{optionName}() {
+    browser.element('[id="{optionId}"]').click().pause(300)
```

Comment:
Fair enough, just if we change the duration of the animation (or change the animation) we'll have to go back and revisit all these.